src/content/third_party_migration/vendors/debian.json

```diff
@@ -0,0 +1,22 @@
+{
+    "vendor": "debian",
+    "description": "Debian",
+    "remotes": [
+        {
+            "url": "https://www.debian.org/security/oval/oval-definitions-{version}.xml",
+            "request": "file",
+            "type": "xml",
```

Comment:
- JSON schema uses the keyword type (a lot). To make the schema easier to develop, debug and read; we should use this keyword the as little as possible.
   - We suggest to change this to `dataFormat` (it also makes this field more descriptive).